test/integration/OrderBook.spec.ts

```diff
@@ -18,7 +18,7 @@ describe('OrderBook', () => {
   before(async () => {
     const config = new Config();
     await config.load();
-    const loggers = Logger.createLoggers();
+    const loggers = Logger.createLoggers(Level.DEBUG);
```

Comment:
For our automated test output, would it make sense to keep the log output limited? We may only be really interested in errors or maybe warnings. Also, I'm thinking we should have a way to not write anything to files for automated test logging, so maybe we also need a flag on whether to log to file or just to console. That can be a separate PR or can be handled here.

Reply:
Do you think of a cli flag or just an argument for `Logger.createLoggers`?

If you mean a cli flag: what about adding a cli flag for the path of the logfile and setting it to `/dev/null` in case you don't want to write to a file?

Reply:
I think we can probably just have an argument in Logger.createLoggers which we'd only set from tests. I do think a configurable logging path would be a good idea too, but `/dev/null` wouldn't work on windows and I'm not sure there's an equivalent. Another option would be to set it to an empty string when we don't want file output, then our code can check for an empty log file path and disable logging to file.

Do you agree with setting the logger level to WARN or ERROR in the tests though?

Reply:
Yeah I agree on setting the logging level to `warn` in the tests. Right now they are way too cluttered imo.

I will take care of the logging path in another PR because this one is already way bigger than I planned.